### ISSUE TITLE
"source ranger" not exiting to desired directory

### ISSUE THREAD

**Issue by CaptainAsbestos** [ranger/ranger#2399]
#### Runtime Environment
OS: Manjaro Linux 5.9.16
Terminal Emulator: Konsole 21.04.3 (bug also happens in Alacritty 0.8.0)
ranger version: ranger-master 
Python version: 3.9.6 (default, Jun 30 2021, 10:22:16) [GCC 11.1.0]
Locale: en_US.UTF-8

#### Current Behavior

When using `source ranger` and exiting the file manager with the q key, the shell remains in the directory where ranger was started, with the following output: 

`bash: cd: $'     1\t/home/user/Music': No such file or directory`
`removed '/tmp/ranger_cd.mTjbcOT4CI'`

#### Expected Behavior

Ranger should exit to the current working directory when q is pressed.

#### Context

While I could just switch to the shell by pressing the s key, I prefer to avoid nesting ranger instances by quitting out of a session rather than pressing control-D only to find out that I didn't in fact have ranger running. Running ranger on my Arch laptop, however, ranger behaves as expected. If I begin ranger in a super-user shell with `su`, ranger behaves as expected.

#### Possible Solutions

I've tried removing my config files, rewriting the config files from scratch, reinstalling ranger with the package manager and cloning and building the code separately, and `source ranger` still exits to wrong directory. 

#### Steps to reproduce
1. begin ranger with `source ranger` or `. ranger`
2. navigate to new directory
3. press q to exit to shell

**Comment by toonn** [comment]
Well, that doesn't look like a proper path to me. Manjaro is a bit notorious for packaging a really old `scope.sh` so maybe the ranger polyglot they package is old too? Try removing it and installing "ranger-fm" from PyPI and see if you get the same problem.

**Comment by CaptainAsbestos** [comment]
Sorry for the late reply, serves me right for not checking my github notifications!

Thank you for the suggestion. I installed with PyPI, problem persisted. The main issue is that, for whatever reason, I think the syntax of `source ranger` prevents me from appending the `--logfile=` flag to see what's going on. I also copied my ranger config files into the superuser's config, and as super user the configs work just like expected. Is there anyway to figure out where that bash output comes from?

Thank you for your patience, I'm little more than an advanced amateur at this.
